index.html

```diff
@@ -12,124 +12,229 @@
+              fill="url(#d)"
+              d="m27.12 15.78-14.61-8.3c-.86-.49-1.62-.43-2.08.06L22.9 20l4.22-4.22Z"
+            />
+            <path
+              fill="#000"
+              d="m27.03 24.13-14.52 8.25c-.81.46-1.54.43-2 .01l-.07.07.07.07c.47.42 1.19.45 2-.01l14.61-8.3-.09-.09Z"
+              opacity=".2"
+            />
+            <path
+              fill="#000"
+              d="M10.44 32.32c-.29-.31-.46-.79-.46-1.4v.15c0 .62.17 1.1.46 1.4l.07-.07-.07-.08ZM32.01 21.3l-4.99 2.83.09.09L32 21.44c.7-.4 1.05-.92 1.05-1.44-.05.47-.4.94-1.04 1.3Z"
+              opacity=".12"
+            />
+            <path
+              fill="#fff"
+              d="m12.51 7.62 19.5 11.08c.63.36.99.82 1.05 1.3 0-.52-.35-1.05-1.05-1.44L12.51 7.48c-1.4-.79-2.54-.13-2.54 1.47v.15c0-1.61 1.15-2.27 2.54-1.48Z"
+              opacity=".25"
+            />
+            <defs>
+              <linearGradient
+                id="a"
+                x1="21.8"
+                x2="5.017"
+                y1="8.71"
+                y2="25.492"
+                gradientUnits="userSpaceOnUse"
+              >
+                <stop stop-color="#00A0FF" />
+                <stop offset=".007" stop-color="#00A1FF" />
+                <stop offset=".26" stop-color="#00BEFF" />
+                <stop offset=".512" stop-color="#00D2FF" />
+                <stop offset=".76" stop-color="#00DFFF" />
+                <stop offset="1" stop-color="#00E3FF" />
+              </linearGradient>
+              <linearGradient
+                id="b"
+                x1="33.834"
+                x2="9.638"
+                y1="20.001"
+                y2="20.001"
+                gradientUnits="userSpaceOnUse"
+              >
+                <stop stop-color="#FFE000" />
+                <stop offset=".409" stop-color="#FFBD00" />
+                <stop offset=".775" stop-color="orange" />
+                <stop offset="1" stop-color="#FF9C00" />
+              </linearGradient>
+              <linearGradient
+                id="c"
+                x1="24.827"
+                x2="2.069"
+                y1="22.296"
+                y2="45.054"
+                gradientUnits="userSpaceOnUse"
+              >
+                <stop stop-color="#FF3A44" />
+                <stop offset="1" stop-color="#C31162" />
+              </linearGradient>
+              <linearGradient
+                id="d"
+                x1="7.297"
+                x2="17.46"
+                y1=".176"
+                y2="10.339"
+                gradientUnits="userSpaceOnUse"
+              >
+                <stop stop-color="#32A071" />
+                <stop offset=".069" stop-color="#2DA771" />
+                <stop offset=".476" stop-color="#15CF74" />
+                <stop offset=".801" stop-color="#06E775" />
+                <stop offset="1" stop-color="#00F076" />
+              </linearGradient>
+            </defs>
+          </svg>
+          <svg class="apps--app-store">
```

Comment:
check this please, where is app store
![image](https://user-images.githubusercontent.com/79144475/200655469-acf10da8-dcb4-402b-9c87-b22d41facf1b.png)